crates/ide/src/hover/tests.rs

```diff
@@ -51,13 +51,17 @@ fn check(ra_fixture: &str, expect: Expect) {
 }
 
 #[track_caller]
-fn check_hover_struct_limit(count: usize, ra_fixture: &str, expect: Expect) {
+fn check_hover_adt_fields_or_variants_limit(
+    count: Option<usize>,
```

Comment:
A neat trick for tests is to take a `impl Into<Option<usize>>` (might need to be `impl Into<Option<u32>>` for fallback reasons). That way you can just use `None` or `5` at call sites instead of `Some(5)`.